packages/backend-data/package.json

```diff
@@ -32,6 +32,8 @@
     "@aws-amplify/backend-output-schemas": "^1.1.0",
     "@aws-amplify/data-construct": "^1.10.1",
     "@aws-amplify/plugin-types": "^1.2.2",
-    "@aws-amplify/data-schema-types": "^1.2.0"
+    "@aws-amplify/graphql-generator": "^0.4.0",
+    "@aws-amplify/data-schema-types": "^1.2.0",
+    "@graphql-codegen/plugin-helpers": "^3.1.1"
```

Comment:
This shouldn't be necessary/helpful once `@aws-amplify/graphql-generator` is released (a hack to align version in my local build)